spec/features/admin/show_spec.rb

```diff
@@ -57,4 +58,16 @@
       expect(page).to_not have_button('Reject Bella')
     end
   end
+
+  describe 'When I click approve button for a pet with multiple applications' do
```

Comment:
Awesome that we just had to add a test since we set up the previous user stories in a great way!